src/pynxtools/nomad/schema.py

```diff
@@ -799,6 +799,7 @@ def __create_package_from_nxdl_directories(nexus_section: Section) -> Package:
                 SubSection(section_def=section, name=section.name)
             )
 
+    package.section_definitions.append(nexus_section)
```

Comment:
Just for future reference. Could you describe how this affects the metainfo generation?

Reply:
added